src/app/campaign/store/actions/account-action.creator.ts

```diff
@@ -1,21 +1,10 @@
-import { Action } from '@ngrx/store';
+import { createAction, props, union } from '@ngrx/store';
 import { ActionTypes } from './action.types';
```

Comment:
v8 NgRx Actions now use createAction to reduce boilerplate and improve type safety. https://medium.com/angular-in-depth/ngrx-action-creators-redesigned-d396960e46da
* action.payload is now obsolete, replaced by props which adds the properties onto the Action
* ActionTypes enum no longer required, in v9 createAction will check plain string values for the action types to be unique. https://twitter.com/tim_deschryver/status/1267849420898476032